src/codecs/gravity/artist.ts

```diff
@@ -0,0 +1,28 @@
+import * as t from "io-ts"
+
+export const GravityArtistCodec = t.type({
+  _id: t.string,
+  id: t.string,
+  alternate_names: t.array(t.string),
+  blurb: t.string,
+  birthday: t.string,
+  collections: t.string,
+  consignable: t.boolean,
+  deathday: t.string,
+  disable_price_context: t.boolean,
+  display_auction_link: t.boolean,
+  forsale_artworks_count: t.number,
+  hometown: t.string,
+  image_url: t.string,
+  image_urls: t.record(t.string, t.string),
+  image_versions: t.array(t.string),
+  location: t.string,
+  name: t.string,
+  nationality: t.string,
+  public: t.boolean,
+  published_artworks_count: t.number,
+  sortable_id: t.string,
+  years: t.string,
+})
```

Comment:
Is there an expression of nullability here? Given that we know some fields (birthday/deathday) may and or may not be returned we definitely don't want the whole thing to error out if they're not present. 

I'd also generally be concerned if we had to replicate a fields nullability in this Codec.

Reply:
Yeah, you'd do something like `t.union([t.string, t.null])`

The funny thing about Gravity is that for fields like birthday, etc, the default is empty string rather than null. So yeah, we'd want to get the typing right for stuff like that: if it can be null *in general* then we need to express that in the codec. Which I assume is the case actually, since none of those are validated?

To shake out the possible shape of the types we'd probably want to do something like bulk run API responses thru them until we know we've got that more or less covered.